### PR TITLE
Windows instructions update + citation update

### LINKED ISSUE #1248: Windows doc installation update is needed for Python 3, Anaconda and VTK support
To successfully install dipy in windows 10 with Python 3.6 I had to:

Find which is the correct Visual C++ compiler for Python 3.6.
Cython needs the exact compiler which was used to create CPython for the specific version of Python.
When you run python in a terminal you can get the MSC_VER and then find the relevant compiler using this list

```
Visual C++ 2008  (9.0)          MSC_VER=1500
Visual C++ 2010 (10.0)          MSC_VER=1600
Visual C++ 2012 (11.0)          MSC_VER=1700
Visual C++ 2013 (12.0)          MSC_VER=1800
Visual C++ 2015 (14.0)          MSC_VER=1900
Visual C++ 2017 (15.0)          MSC_VER=1910
```

In my case it was MSC_VER=1900 so I installed visual build tools 2015. See more info about that here:

https://wiki.python.org/moin/WindowsCompilers
http://landinghub.visualstudio.com/visual-cpp-build-tools

For vtk 7.1.0 conda-forge or conda do not have vtk 7.1.0 for Python 3. However clinicalgraphics provide a working version.

``conda install -c clinicalgraphics vtk=7.1.0
``

### PR DESCRIPTION
**Pull request by skoudoro**
The PR is just an update of windows instructions.

This should resolve #1248, #1084, #1146.

I use this opportunity to add an issue template

### CODE REVIEW

File: doc/cite.rst [hunk 1]
```diff
@@ -19,6 +19,14 @@ Publications
...
 Neuroscience, 6 (175).
 
+[9] Garyfallidis E, Cote M-A, Rheault F, Sidhu J, Hau J, Petit L, Fortin D, Cunanne S, Descoteaux M,  `Recognition of white matter bundles using local and global streamline-based registration and clustering. <http://www.sciencedirect.com/science/article/pii/S1053811917305839>`
+
+[10] Garyfallidis E, Ocegueda O, Wassermann D, Descoteaux M. `Robust and efficient linear registration of white-matter fascicles in the space of streamlines. <http://www.sciencedirect.com/science/article/pii/S1053811915003961>`
+
+[11] Rokem A, Yeatman JD, Pestilli F, Kay KN, Mezer A, et al. (2015), `Evaluating the Accuracy of Diffusion MRI Models in White Matter. <http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0123272>`
+
+[12] Ocegueda O, Dalmau O, Garyfallidis E, Descoteaux M, Rivera M, `On the computation of integrals over fixed-size rectangles of arbitrary dimension. <http://www.sciencedirect.com/science/article/pii/S0167865516300861>`
+
```
Comment (arokem) [inline doc/cite.rst]: Please add: [Re] Optimization of a free water elimination two-compartment model for diffusion tensor imaging, Rafael Neto Henriques, Ariel Rokem, Eleftherios Garyfallidis, Samuel St-Jean, Eric Thomas Peterson, Marta Morgado Correia, ReScience volume 3, issue 1, article number 2, 2017.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+## Way to reproduce
+[If reporting a bug, please include the following important information:]
+- [ ] Code example
+- [ ] Relevant images (if any)
+- [ ] Operating system and version (run `python -c "import platform; print(platform.platform())"`)
+- [ ] Python version (run `python -c "import sys; print("Python", sys.version)"`)
+- [ ] dipy version (run `python -c "import dipy; print(dipy.__version__)"`)
+- [ ] dependency version (numpy, scipy, vtk)
```
Comment (arokem) [inline ISSUE_TEMPLATE.md]: Do we want to know versions of other dependencies or just these three?

File: doc/cite.rst (hunk 1 above)
Comment (skoudoro) [reply doc/cite.rst]: done ! -> [e55e20a](https://github.com/nipy/dipy/pull/1361/commits/e55e20a74d26a8f151da7cc335af7049394b714e)

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (skoudoro) [reply ISSUE_TEMPLATE.md]: I suppose they are the most important, but I add the others dependencies to get the complete information ([last commit](https://github.com/nipy/dipy/pull/1361/commits/0c2dc2ae7f1a463503f83613e04ce777ef05a542))

File: doc/installation.rst [hunk 3]
```diff
@@ -311,7 +326,13 @@ or the CLANG compiler. This depends on your python installation:
 Under Anaconda
 ~~~~~~~~~~~~~~~~
 
-If you are using Anaconda, you will need to use GCC. Run the following::
+If you are using Anaconda, you will need to use GCC. THe first option is to run the following command::
+
+    conda install gcc
+
+After this installation, gcc will be your default compiler in anaconda environment.
```
Comment (Garyfallidis) [inline doc/installation.rst]: Anaconda

**Comment by Garyfallidis** [comment]
Okay, looks ready. Release day. Merging...

**Comment by Garyfallidis** [comment]
Looks, good. Release day! Merging!

**Pull request merged**